python/rascaline-torch/rascaline/torch/calculator_base.py

```diff
@@ -56,15 +56,20 @@ class CalculatorModule(torch.nn.Module):
     """
 
     def __init__(self, name: str, parameters: str):
-        """"""
```

Comment:
I remember we need the empty docstring here for the docs to render right, I'll check again